ambry-api/src/main/java/com/github/ambry/clustermap/ClusterParticipant.java

```diff
@@ -48,6 +49,19 @@ void participate(List<AmbryStatsReport> ambryStatsReports, AccountStatsStore acc
    */
   boolean setReplicaSealedState(ReplicaId replicaId, boolean isSealed);
 
+  /**
+   * Set or reset the partial-sealed state of the given replica.
+   * @param replicaId the {@link ReplicaId} whose sealed state will be updated.
+   * @param isPartiallySealed if true, the replica will be marked as partially sealed. Replicas marked as partially
```

Comment:
nit:
1. unrelate it to reserved meta data.
2. What if isPartiallySealed is false? what's the outcome?

Reply:
fixed and updated.